pkg/skaffold/event/event.go

```diff
@@ -677,6 +730,15 @@ func ResetStateOnBuild() {
 	handler.setState(newState)
 }
 
+// ResetStateOnTest resets the test, deploy, sync and status check state
+func ResetStateOnTest() {
+	newState := handler.getState()
+	newState.TestState.Status = NotStarted
+	newState.DeployState.Status = NotStarted
+	newState.DeployState.StatusCode = proto.StatusCode_OK
```

Comment:
do not reset `Deploy` relates state.
```suggestion
	newState.TestState.Status = NotStarted
```

Reply:
Removed deploy state updates in  ResetStateOnTest().